packages/web-js/src/models/CorbadoAppState.ts

```diff
@@ -78,10 +78,6 @@ export class CorbadoAppState {
   }
 
   logout() {
-    if (!this.corbadoApp) {
```

Comment:
why are we removing this?

Reply:
this was a redundant check. We already do this in the parent class. Since, I need to make a change to create a new version and test my work I removed this redundant check.